### PR TITLE
add metrics

### PR DESCRIPTION
**Pull request by rudigiesler**
use the metrics helper to add metrics: https://www.npmjs.org/package/go-jsbox-metrics-helper

### CODE REVIEW

File: go-app.js [hunk 1]
```diff
@@ -15,12 +15,59 @@ go.app = function() {
...
+        // application
+            mh = new MetricsHelper(self.im);
+            mh
+                // Total unique users
+                .add.total_unique_users()
+                // Total and weekly USSD sessions
+                .add.total_sessions()
+                // Total and weekly completed reports
+                // Average sessions per complete report
+                .add.trigger({state: 'states:send-report',action: 'enter'}, {
```
Comment (hodgestar) [inline go-app.js]: Missing space after comma.

**Comment by hodgestar** [comment]
Other than tiny missing-space comment, :+1:.

I really like how we can now define all the metrics in one place rather than scattering them around the code. We should show this example to @smn and @imsickofmaps.

**Comment by hodgestar** [comment]
Oh, and we should run through this with Pippa and Lauren so that they know what their metrics mean under the hood.

**Comment by smn** [comment]
holy smokes this is awesome! \o/

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**